### DISCUSSION TITLE
Is the cookie consentframework.com trustworthy?

### DISCUSSION THREAD

**Discussion by Ray3333333** [uBlockOrigin/uAssets#13041] [Q&A]
I think I was able to block it in Firefox in Sandboxie to see if I could block it using Firefox with success, but then I couldn't answer if I should be doing that? because I don't know if consentframework.com is needed for a ublock filter to work. I've been figuratively tearing my hair out to find out why it keeps popping up in cookies. I even visited that site which seems to allow me to opt out of selling my personal data, but because I have Firefox delete all history when closing Firefox I think that may be why, and also because I use Sandboxie most of time which is the way I prefer it. This is not my place of expertise. I made sure to go to that website to opt out of selling my personal data outside of Sandboxie, but it will not remember because of deleted Firefox history on closing Firefox I think..

So is consentframework.com suppose to be trustworthy?, Or is it bad? I don't think I've seen it before. 

I should perhaps ask instead if ublock requires consentframework.com to work for blocking bad ads and so on?

**Comment by u-RraaLL** [comment]
> if ublock requires consentframework.com to work

No, uBO doesn't require any domains to work (other than when updating filter lists).

No idea what that domain is or whether or not it's trustworthy.

Can you explain where you run into it originally? Why would you associate it with uBO in the first place?

**Reply by Ray3333333** [reply]
After I looked it up a whole lot, I read that Google is rolling it out or something like that, but it might not become a cookie that takes off. So it appears that it is most likely trustworthy, because I haven't read anything bad about it only good, yet when searching for it using Google, but I don't know.

The reason I thought it was related to ublock was I found it in the code of one of the fliter lists called "ublock filters" which is right above the filter called "ublock filters - badware risks" What is weird is that that doesn't have " - " after it with a description. It is also right below "My Filters".

But when I read the code to me it didn't make sense whether it was supposed to blocked or allowed. It was under Yavli ads. I didn't understand how to use ublock code to customize for blocking an ad using the "My Filter", but I was able to block that cookie in "My Filters", which I have never done before, but I don't understand the code used in the filter that use much more than just a url link. I only made a filter with just the url for that cookie. Anyways, sorry for any trouble.

**Reply by u-RraaLL** [reply]
> What is weird is that that doesn't have " - " after it with a description

That's not weird. It's the main filter list. The description is inside it. The rest of uBO's own list have a `- something` to explain what purpose they serve and to easier arrange new commits to a proper list.

> But when I read the code to me it didn't make sense whether it was supposed to blocked or allowed.

`denyallow` means the domain is to be allowed, while other (not listed) domains are to be blocked. [More on that here](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#denyallow)

> I was able to block that cookie in "My Filters", which I have never done before, but I don't understand the code used in the filter that use much more than just a url link. I only made a filter with just the url for that cookie.

Do you mean a cookie prompt?

Anyway, to understand more about filtering, other than the section I directly linked to  - go ahead and read the whole page. You can supplement that with the links that are right there at the top and lead to the adblock and adguard filtering guides.

**Reply by Ray3333333** [reply]
Great!, that makes sense. Now I think I understand the code better.

Do you mean a cookie prompt?
As in intrusive?, No, but I just wanted to know if it was a safe cookie, which it very much appears to be safe, because it was on the list as allow among other cookies.

Thank you very much for the guidance.

**Comment by ellamacrow98** [comment]
I don't[ ](https://www.copilotsearch.com/)think so.

**Reply by Ray3333333** [reply]
What do you mean?, That you think it is trustworthy?, Or not?, Or that you are not sure, and that might be a good reason not to trust them?

**Comment by MasterKia** [comment]
> I even visited that site which seems to allow me to opt out of selling my personal data

> I made sure to go to that website to opt out of selling my personal data 

Read gorhill's answer:

https://www.reddit.com/r/uBlockOrigin/comments/qlj2kl/will_ublock_origin_send_the_gpc_donotsell_signal/

**Reply by Ray3333333** [reply]
What I said to do works, but for me what you said you did did not work for me. After opting out, if I delete Firefox history it will just pop right back up in cookies as if I did nothing. Also I use Sandboxie and have it delete the contents of Sandboxie when closing(terminating) a sandbox to have a cleaner Firefox, so opting out from there for me is not an option as far as I can tell.

**Reply by MasterKia** [reply]
> opting out from there

Data collection should be **_opt-in_**, not opt-out.
> There is no way for users to find out whether a site respects this or not, so safest assumption is that it's not respected, just like `DoNotTrack` was not.

**Reply by Ray3333333** [reply]
I think I misunderstood what you are suggesting. Can you give me a more direct answer as to what to do?

**Reply by MasterKia** [reply]
What are you aiming for?

**Reply by Ray3333333** [reply]
OK, I think I get it now. I was trying to make my last comment before you just made another comment. So there there is no other sollution then I guess.

**Reply by Ray3333333** [reply]
I have already blocked it, so I guess it doesn't matter. Just look at what I have written. Hostfile, block it in ublock, yadda, yadda yadda. Sorry for all the ranting.

**Comment by 1996haseeb** [comment]
